### DISCUSSION TITLE
Not all songs are played

### DISCUSSION THREAD

**Discussion by Pukesmiley** [dtcooper/raspotify#638] [General]
Raspotify only playes the first 85+- Songs from a playlist and then stops. Even when starting in the middle by hand and then starting shuffle the songs where chosen from those inbetween nr 1328 and 1246 (difference is 82) maybe the range is a little higher, but in a playlist of 2700 songs this shouldnt happen

**Comment by giggywithit** [comment]
How did you set the settings?\
Are you caching?
